public/404.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="icon" type="image/x-icon" href="/images/dailycat-logo.svg" />
+    <link rel="stylesheet" href="/stylesheets/404Style.css" />
+    <link rel="stylesheet" href="/stylesheets/fonts.css" />
+    <title>Dailycat</title>
+  </head>
+  <body>
+    <nav id="navbar">
+      <h1 id="page-title">Dailycat</h1>
```

Comment:
Oh yeah, this was the correct `h1`. Nice.

Reply:
ur cringe